spec/models/misc_item_spec.rb

```diff
@@ -251,106 +253,81 @@
       let(:item) { create(:misc_item, name: "Wylandria's Soul Gem") }
 
       it "doesn't set the association" do
-        item.validate
+        validate
         expect(item.canonical_misc_item).to be_nil
       end
     end
 
-    context "when multiple complete matches can't be further differentiated" do
-      let(:game) { create(:game) }
-      let(:item) { build(:misc_item, name: 'Skull', unit_weight: 2, game:) }
+    context 'when updating in-game item attributes' do
+      let(:item) { create(:misc_item, :with_matching_canonical) }
 
-      context 'when a canonical model indicates a unique item' do
```

Comment:
Don't worry, this functionality is still tested elsewhere in the file. These tests were ~duplicates.